extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -62,6 +62,13 @@
         @ConfigItem(defaultValue = "true")
         public boolean enabled;
 
+        /**
+         * If {@code true}, the camel context will be started at initialization time
+         * to boot up faster.
+         */
+        @ConfigItem
```

Comment:
boot up faster is better than boot up slower. Shouldn't the default be `true`?

Reply:
Yes, but see my comment above.

Reply:
IIRC the lightweight context has some limitation also in term of EIPs so it should be a conscious decision (unless we remove such limitations of course)

Reply:
> has some limitation also in term of EIPs

I'd very much like to see all such limitations documented.